### PR TITLE
[DNM] raftstore: add lock cf to large cfs

### PR DESCRIPTION
**Pull request by sticnarf**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note

### CODE REVIEW

File: components/engine_traits/src/cf_defs.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ pub const CF_LOCK: CfName = "lock";
 pub const CF_WRITE: CfName = "write";
 pub const CF_RAFT: CfName = "raft";
 // Cfs that should be very large generally.
-pub const LARGE_CFS: &[CfName] = &[CF_DEFAULT, CF_WRITE];
+pub const LARGE_CFS: &[CfName] = &[CF_DEFAULT, CF_LOCK, CF_WRITE];
```
Comment (yiwu-arbug) [inline components/engine_traits/src/cf_defs.rs:9]: just curious, how does large CFs being handle differently?

Comment (sticnarf) [reply components/engine_traits/src/cf_defs.rs:9]: The estimated size of `LARGE_CFS` are used to determine whether to split a region.

Comment (zhangjinpeng1987) [reply components/engine_traits/src/cf_defs.rs:9]: I guess in the huge transaction scenario the prewrite stage will write many data into lock cf, and this should trigger region split.

**Comment by zhangjinpeng1987** [comment]
Because lock cf doesn't have table properties in sst file, so maybe split check will scan the lock cf to get the approximate size.

**Comment by nrc** [comment]
Closing in favour of https://github.com/tikv/tikv/pull/7676

**Pull request closed without merging**